lib/clearance/password_strategies/bcrypt_migration_from_sha1.rb

```diff
@@ -22,10 +29,12 @@ def initialize(user)
       end
 
       def authenticated?(password)
+        warn "#{Kernel.caller.first}: #{DEPRECATION_MESSAGE}"
```

Comment:
If I'm not mistaken, the choice is between a heredoc and proper indentation? I'd rather have the indentation. Any other options?